aws/resource_aws_msk_cluster_test.go

```diff
@@ -515,6 +515,44 @@ func TestAccAWSMskCluster_LoggingInfo(t *testing.T) {
 	})
 }
 
+func TestAccAWSMskCluster_KafkaVersionUpdate(t *testing.T) {
+	var cluster1, cluster2 kafka.ClusterInfo
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_msk_cluster.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSMsk(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMskClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMskClusterConfigKafkaVersion(rName, "2.2.1"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMskClusterExists(resourceName, &cluster1),
+					resource.TestCheckResourceAttr(resourceName, "kafka_version", "2.2.1"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"bootstrap_brokers",     // API may mutate ordering and selection of brokers to return
+					"bootstrap_brokers_tls", // API may mutate ordering and selection of brokers to return
+				},
+			},
+			{
+				Config: testAccMskClusterConfigKafkaVersion(rName, "2.4.1"),
```

Comment:
Note that after September 23rd, this version will no longer be available – see [this announcement](https://aws.amazon.com/about-aws/whats-new/2020/09/amazon-msk-now-offers-version-2-4-1-1-fixing-a-perpetual-rebalance-bug-in-apache-kafka-2-4-1/).

```suggestion
				Config: testAccMskClusterConfigKafkaVersion(rName, "2.4.1.1"),
```

Reply:
thanks, fixed it.